docs/authentication/saml/account-linking.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking during SAML SSO.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both SAML SSO and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an SAML provider returns an email address that matches an existing account, assuming a single owner for each email address.
+
+## How it works
+
+When a user attempts to sign in or sign up, Clerk first checks the email address provided. Clerk will attempt to link the SAML account with any existing Clerk account that shares the same email address.
```

Comment:
```suggestion
When a user attempts to sign in or sign up, Clerk first checks the provided email address. Clerk will attempt to link the SAML account with any existing Clerk account that shares the same email address.
```

Is this scoped within Clerk apps, or is this referring to Clerk accounts?

To clarify, which of these is the case?
A. Clerk is going to merge _my_ account as a developer if I sign in with SSO to Clerk.com?
B. Clerk is going to merge my _users'_ accounts if they sign into my Clerk app with SSO?